beta-architecture/diagrams/tdr-beta-high-level-architecture.svg

```diff
@@ -0,0 +1 @@
.05,0,0)" xlink:href="#aY"/><use transform="matrix(0.05,0,0,0.05,9.600000000000001,0)" xlink:href="#aZ"/><use transform="matrix(0.05,0,0,0.05,19.6,0)" xlink:href="#ba"/><use transform="matrix(0.05,0,0,0.05,26.6,0)" xlink:href="#ba"/><use transform="matrix(0.05,0,0,0.05,33.6,0)" xlink:href="#bb"/><use transform="matrix(0.05,0,0,0.05,43.6,0)" xlink:href="#bc"/><use transform="matrix(0.05,0,0,0.05,49.550000000000004,0)" xlink:href="#bd"/><use transform="matrix(0.05,0,0,0.05,60.5,0)" xlink:href="#ba"/><use transform="matrix(0.05,0,0,0.05,67.5,0)" xlink:href="#be"/><use transform="matrix(0.05,0,0,0.05,83.5,0)" xlink:href="#bf"/></g><path d="M137-138c1-29-70-34-71-4 15 46 118 7 119 86 1 83-164 76-172 9l43-7c4 19 20 25 44 25 33 8 57-30 24-41C81-84 22-81 20-136c-2-80 154-74 161-7" id="bg"/><path d="M115-3C79 11 28 4 28-45v-112H4v-33h27l15-45h31v45h36v33H77v99c-1 23 16 31 38 25v30" id="bh"/><g id="c"><use transform="matrix(0.05,0,0,0.05,0,0)" xlink:href="#bg"/><use transform="matrix(0.05,0,0,0.05,10,0)" xlink:href="#bh"/><use transform="matrix(0.05,0,0,0.05,15.949999999999998,0)" xlink:href="#bb"/><use transform="matrix(0.05,0,0,0.05,25.950000000000003,0)" xlink:href="#bh"/><use transform="matrix(0.05,0,0,0.05,31.899999999999995,0)" xlink:href="#aZ"/></g><g id="d"><use transform="matrix(0.05,0,0,0.05,0,0)" xlink:href="#aY"/><use transform="matrix(0.05,0,0,0.05,9.600000000000001,0)" xlink:href="#aZ"/><use transform="matrix(0.05,0,0,0.05,19.6,0)" xlink:href="#ba"/><use transform="matrix(0.05,0,0,0.05,26.6,0)" xlink:href="#ba"/><use transform="matrix(0.05,0,0,0.05,33.6,0)" xlink:href="#bb"/><use transform="matrix(0.05,0,0,0.05,43.6,0)" xlink:href="#bc"/><use transform="matrix(0.05,0,0,0.05,49.550000000000004,0)" xlink:href="#bd"/><use transform="matrix(0.05,0,0,0.05,60.5,0)" xlink:href="#ba"/><use transform="matrix(0.05,0,0,0.05,67.5,0)" xlink:href="#be"/><use transform="matrix(0.05,0,0,0.05,83.5,0)" xlink:href="#bf"/></g><path d="M25 0v-261h50V0H25" id="bi"/><path d="M190-63c-7 42-38 67-86 67-59 0-84-38-90-98-12-110 154-137 174-36l-49 2c-2-19-15-32-35-32-30 0-35 28-38 64-6 74 65 87 74 30" id="bj"/><path d="M147 0L96-86 75-71V0H25v-261h50v150l67-79h53l-66 74L201 0h-54" id="bk"/><g id="e"><use transform="matrix(0.05,0,0,0.05,0,0)" xlink:href="#bg"/><use transform="matrix(0.05,0,0,0.05,10,0)" xlink:href="#bh"/><use transform="matrix(0.05,0,0,0.05,15.949999999999998,0)" xlink:href="#bb"/><use transform="matrix(0.05,0,0,0.05,25.950000000000003,0)" xlink:href="#bh"/><use transform="matrix(0.05,0,0,0.05,31.899999999999995,0)" xlink:href="#aZ"/><use transform="matrix(0.05,0,0,0.05,41.9,0)" xlink:href="#bi"/><use transform="matrix(0.05,0,0,0.05,46.9,0)" xlink:href="#bd"/><use transform="matrix(0.05,0,0,0.05,57.85,0)" xlink:href="#bj"/><use transform="matrix(0.05,0,0,0.05,67.85,0)" xlink:href="#bk"/></g><path d="M24-248c120-7 223 5 221 122C244-46 201 0 124 0H24v-248zM76-40c74 7 117-18 117-86 0-67-45-88-117-82v168" id="bl"/><g id="f"><use transform="matrix(0.074074074
```

Comment:
Thanks for adding the labels. I think it makes the dependencies between components much clearer. :+1: 

What do you think about switching all the double-headed arrows for single-direction ones? I think it would help to make it clear that (for example) the "Add Series" task calls the API server, not the other way around.

Reply:
Resolved